GMG/Scenes/ChordProgress/ChordProgressIntent.swift

```diff
@@ -107,6 +110,30 @@ final class ChordProgressIntent: ChordProgressIntentProtocol {
 
         scorePlayer.pause()
         scorePlayer.seek(to: time)
+
+        if scorePlayer.wasPlayingBeforeLastCommand {
+            scorePlayer.play()
+        }
```

Comment:
여기도 `scorePlayer.seek(to: time)`만 사용하면 자연스럽게 이전 재생 상태에 따라 작동됩니다.

Reply:
네 
```suggestion
        scorePlayer.seek(to: time)
```

이렇게만 하면 되겠네요!